### PR TITLE
[ADD] udes_stock: Patching cursor not to commit changes

### PR DESCRIPTION
**Pull request by armandcela**
SE-1104
Signed-off-by: Armand Cela <armand.cela@unipart.io>

### CODE REVIEW

**Review by rajeeshknambiar — APPROVED** [review]

**Comment by PeterAlabaster** [comment]
nit: commit typo `cursour` -> `cursor`

**Comment by PeterAlabaster** [comment]
we may want to tidy cursor patches in here: addons/udes_stock_cron/tests/test_stock_picking.py

File: addons/udes_stock/tests/common.py [hunk 1]
```diff
@@ -563,7 +564,15 @@ def update_move_lines(cls, move_lines, qty=None, user=None):
 class BaseUDES(UnconfiguredBaseUDES):
     @classmethod
     def setUpClass(cls):
+        """
+        Patching the cursor in order not to commit if in code there are slf.env.cr.commit
```
Comment (kdwyer) [inline addons/udes_stock/tests/common.py:568]: typo s/slf/self

Comment (armandcela) [reply addons/udes_stock/tests/common.py]: I noticed that too :)

**Review by PeterAlabaster — APPROVED** [review]

**Pull request merged**